api/actions/createWorkItem.js

```diff
@@ -67,28 +79,39 @@ module.exports = {
                                 interval: 60000
                             });
                             if (syncedItem) {
-                                return await Github.createComment({
-                                    req,
-                                    body: `This issue has been linked to a new work item: ${getWorkItemUrl(
-                                        syncedItem,
-                                        hideWorkItemUrl
-                                    )}`
-                                });
+                                return await updateIssue(req, `This issue has been linked to a new work item: ${getWorkItemUrl(syncedItem, hideWorkItemUrl)}`, suppressGithubComments);
                             }
-                            return await Github.createComment({
-                                req,
-                                body:
-                                    'Sorry we could wait until Heroku connect make the syncronization.'
-                            });
+                            return await updateIssue(req, 'Sorry we could wait until Heroku connect make the synchronization.', suppressGithubComments);
                         }
                     }
                 );
             }
-            return await Github.createComment({
-                req,
-                body: getBuildErrorMessage(config, milestone)
-            });
+            return await updateIssue(req, getBuildErrorMessage(config, milestone, suppressGithubComments));
         }
         return null;
     }
 };
+function formatToGus(url, body) {
+    var formattedDescription;
+    remark().use(markdown).use(strip).process(body, (err, file) => {
+        if (err) {
+            throw err;
+        }
+        formattedDescription = 'Github issue link: '.concat(url, '\n', String(file));
```

Comment:
Adds the github link at the top of the description. We can make it a table as @reiniergs suggested. But will involve more work and I can take that up after finishing up the changelist sync story I'm currently working.